.travis.yml

```diff
@@ -21,7 +21,6 @@ services:
   - postgresql
   - redis-server
   - docker
-  - mongodb
```

Comment:
we need to keep this line here. it's not part of the plugin template, but we need it.

Reply:
@dkliban I re-added that line.